src/modules/FioAddress/util.js

```diff
@@ -99,6 +101,24 @@ const setConnectedWalletsFromFile = async (fioWallet: EdgeCurrencyWallet, fioAdd
   }
 }
 
+const getFioExpiredCheckFromDislket = async (disklet: Disklet): Promise<Date> => {
```

Comment:
Call this functions once on login, then put the value in redux so that next time you can get it from redux and not from the disk